### ISSUE TITLE
CDDA wont maximize with 'windowed borderless'

### ISSUE THREAD

**Issue by itsdlow** [CleverRaven/Cataclysm-DDA#55096]
### Describe the bug

- On launching CDDA through "xwin server", the application flashes across dual-monitor 'extends' setup (resolution stretched across both monitors) 
- And then, the application instantly minimized
- Upon attempt to maximize application, through taskbar, the application again flashed across my monitors and then minimized.

### Steps to reproduce

1. Followed download instructions on [this page](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/COMPILING/COMPILING-CYGWIN.md)
2.  Followed 'run' instructions on [this page](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/COMPILING/COMPILING-CYGWIN.md#running)

### Expected behavior

I expected CDDA to open correctly (maximized) upon launch through XWin server + UX Term

### Screenshots

N/A

### Versions and configuration

Game Version: 0.F-5745-g538ff36f96
OS: Windows 10
Graphics: tiles
Ingame language: English
Mods: N/A

### Additional context

I was able to successfully maximize CDDA after changing the settings of the game from "windowed borderless"
- I was able to solve the problem by changing my settings, i was able to acheieve this without being able to maximize the application due to CDDA registering my keyboard input, despite not being maximized.
- And would continue reading input so long as i did not click elsewhere. (after attempting to maximize)

- In summary, i believe the problem can be avoided by having the default be any setting besides "windowed borderless"
